core/producer_server_process.py

```diff
@@ -95,7 +95,8 @@ class ProducerServerProcess:
     """
 
     def __init__(self, ip: str, port: int):
-        """
+        """Initialize object of ProducerServerProcess to ip and port parameters.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)